app/models/feed.rb

```diff
@@ -41,11 +41,12 @@ class Feed < ApplicationRecord
 
   validates :slug, uniqueness: {scope: :podcast_id}, if: :podcast_id?
   validates_format_of :slug, allow_nil: true, with: /\A[0-9a-zA-Z_-]+\z/
-  validates_format_of :slug, without: /\A(images|\w{8}-\w{4}-\w{4}-\w{4}-\w{12})\z/
+  validates_format_of :slug, without: /\A(default|images|\w{8}-\w{4}-\w{4}-\w{4}-\w{12})\z/
   validates :file_name, presence: true, format: {with: /\A[0-9a-zA-Z_.-]+\z/}
   validates :include_zones, placement_zones: true
   validates :audio_format, audio_format: true
   validates :title, presence: true, unless: :default?
+  validates :episode_offset_seconds, numericality: {equal_to: 0}, allow_nil: true, if: :default?
```

Comment:
To keep things "simple" ... I decided to restrict default_feeds to _not_ have any episode offset.  Episodes must publish at exactly their `published_at` in the default feed.

Which is currently the case in prod, and I can't imagine why you'd want to offset the default feed?  It would be confusing.

Reply:
Now that negative offsets work correctly ... I think it makes the most sense for the default feed to always be at 0, and the private feed to be `-604800` (7 days early).

Reply:
I think this is 6 of one, -6 of the other, so I have no strong objection here - it seems weird to me, but let's wait and see if we got a podcaster that wants to do this, and we can see what they think, and this is good until such feedback